renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -435,12 +435,7 @@ __attribute__((visibility("default"))) EGLContext eglCreateContext(EGLDisplay di
   init.depthBits = value;
   eglhooks.eglGetConfigAttrib_real(display, config, EGL_STENCIL_SIZE, &value);
   init.stencilBits = value;
-// TODO: how to detect this?
-#if ENABLED(RDOC_ANDROID)
   init.isSRGB = 0;
```

Comment:
Ill try fixing it now: check for the EGL attrib as well as 32bit buffer size. Hold off on the commit.